resources/dicts/relationship_events/special_character.json

```diff
@@ -29,7 +29,6 @@
     ],
     "calm": [
       "relaxing with (cat).",
-      "is soothing (cat)'s irrational thoughts.",
```

Comment:
Yes, I forgot include the reason why on my PR. This was removed because the calm personality relationship events are heavily loaded to be emotional labor with this and the 'is helping (cat) calm down." 

I made the decision to remove it and leave the less emotionally charged event in, but I can re-add it if needed.

Reply:
I think it would be better to add, rather than remove.  Perhaps coming up with some less emotional calm events would be preferable.

Reply:
I'll add that back in right now then.